SETUP/tests/misc_ParamValidatorTest.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-class ParamValidatorTest extends PHPUnit\Framework\TestCase
+class misc_ParamValidatorTest extends PHPUnit\Framework\TestCase
```

Comment:
The original intent behind the `misc_` convention was to indicate what files the tested functionality was in. But we aren't really doing that, so I'd much rather just rename the files to not have the `misc_` than to rename the class to include it.

Because I assume your goal is to have the class names match the filename?

Reply:
That's right. phpunit doesn't seem happy if you don't.

Reply:
Done.